README.md

```diff
@@ -10,7 +10,10 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 
 ```js
 
-var superagent = require('superagent-use');
+var superagent = require('superagent');
+/* Pass the superagent object to superagent-use
+   in order to be provided with new `use` functionality */
+require('superagent-use')(superagent);
```

Comment:
But I understand that `object-assign` brings in a new dependency, but I think [`extend`](https://www.npmjs.com/package/extend) should work just as fine, which is already a dependency of superagent itself.